janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -0,0 +1,190 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.diskstorage.cql;
+
+import org.janusgraph.diskstorage.configuration.ConfigElement;
+import org.janusgraph.diskstorage.configuration.ConfigNamespace;
+import org.janusgraph.diskstorage.configuration.ConfigOption;
+import org.janusgraph.graphdb.configuration.GraphDatabaseConfiguration;
+import org.janusgraph.graphdb.configuration.PreInitializeConfigOptions;
+
+/**
+ * Configuration options for the CQL storage backend. These are managed under
+ * the 'cql' namespace in the configuration.
+ */
+@PreInitializeConfigOptions
+public interface CQLConfigOptions {
+
+    public static final ConfigNamespace CQL_NS = new ConfigNamespace(
+            GraphDatabaseConfiguration.STORAGE_NS,
+            "cql",
+            "CQL storage backend options");
+
+    public static final ConfigOption<String> KEYSPACE = new ConfigOption<>(
```

Comment:
Generated configuration reference documentation page does not appear to include a section for `storage.cql` with these options. It should be automatically generated I think (see [configref.txt#L6](https://github.com/JanusGraph/janusgraph/blob/8914caed850b9d641c8e5b4e15c20f839f6d076b/docs/configref.txt#L6)).